docs/reference/ingest/processors/enrich.asciidoc

```diff
@@ -6,95 +6,16 @@
 The `enrich` processor can enrich documents with data from another index.
 See <<ingest-enriching-data,enrich data>> section for more information how to set this up and
 check out the <<enrich-processor-getting-started,getting started>> to get familiar with enrich policies and related APIs.
-a
+
 [[enrich-options]]
 .Enrich Options
 [options="header"]
 |======
 | Name               | Required  | Default              | Description
 | `policy_name`      | yes       | -                    | The name of the enrich policy to use.
-| `enrich_key`       | no        | Policy enrich_key    | The field to get the value from for the enrich lookup.
+| `field`            | yes       | -                    | The field to get the value from for the enrich lookup.
+| `target_field`     | yes       | -                    | The field that will hold the content of the looked up document as json object.
```

Comment:
How about : 
`The field that will hold the content of the looked up document as json object.`
to
`The field that will be used for the enrichment data`